### ISSUE TITLE
QuickCapture: Bug with opening web map / basemap when transitioning from WiFi to Cellular

### ISSUE THREAD

**Issue by pjdohertygis** [pjdohertygis/WideAreaSearchTemplates#130]
We are working through an issue (multiple issues?) with Esri. When a user goes to the basemap, especially when editing the location of a point - the map either times out or says you do not have access. 

Known issue v1.9 - short-term fix, seems related to leaving wifi turned on and then walking away. Also, this can be "resolved" by closing the app an re-opening. 

Potentially fixed in v1.10.

Teams that hit this issue.
- MD-TF1
- CN
- CA-TF7
- Paul using his FEMA Phone

**Comment by pjdohertygis** [comment]
Here is feedback from CA-TF7 

Hi Paul, 

I would be more than happy to help out with troubleshooting. 

Situation- over a three-day period we had six drills on data collection. We were using the iPad mini 5, model number MUY82LL/A, and ArcGIS QuickCapture version 1.9.127. Approximately 150 people used these devices to collect data. We started in the classroom; I instructed all users to turn off the wifi, we left the building started a search track, walked about 1000' to a track. I had set up 20 pictures of disasters and attached them to a chain link fence. Each picture was approximately 15' from each other. Each picture had information for the students mark waypoints and preform data collection. Each class had +/- 20 students. After the data collection we walked back to the classroom and viewed the collected information on the dashboard. 

Issues:
•	On occasions, when using the pencil and trying to zoom into a location the app would freeze. We were unable to move out of the map area and would have to double click the circle on the device and close the app. This happened numerous times, almost at each drill. 
•	After relaunching the app. We lost the tracks. I looked at the dashboard and it appeared that the tracks were not saved after the app froze. 
•	Some of the students forgot to click on the search icon again after the search was completed and then closed the app. I believe those tracks were lost. 
Hope this helped. let me know if you need anything else. Thanks.

**Comment by IsmaelInRedlands** [comment]
@pjdohertygis  I think we have pinned down some of this basemap issues with 1.10.  We are using a more modern version of the Runtime SDK and things seem to have improved.   I suggest we give the Beta a go if we can although I understand this could be logistically difficult as people will need to install the Beta version from Test Flight.
Ideally, we should do the test once we have support in 1.10 for multiple project user inputs. I estimate we are about 2 weeks from that point.

Even with 1.9, tracks should have not been lost. Even if the app crashes, your tracks are saved. The problem may be that the tracks are in the device. If the project is opened again, the data should automatically be sent.

**Comment by pjdohertygis** [comment]
@IsmaelInRedlands thank you! I did some testing with v1.10 (iPhone) last night and it seemed to reduce the likelihood of the map hanging. At most one time I got the "you're offline" message when transitioning from Wifi to 4G, but it never hung and allowed me to see the map eventually.

We are going to see if we can get some v1.10 beta testing done with the SONIMs from MD-TF1 and iPad minis from CA-TF7.


 ![image](https://user-images.githubusercontent.com/4256979/122483646-11c8f900-cfa1-11eb-9654-0d6621dce737.png)
